### DISCUSSION TITLE
Unable to transfer domain from Google domain to Vercel.

### DISCUSSION THREAD

**Discussion by jessi0419kim** [vercel/community#3054] [Help]
### Summary

When trying to transfer my domain from Google Domain to Vercel, it keeps failing with the error message below.

"The provided auth code does not match with the one expected by the current registar"

I certainly entered exact auth code provided on Google domain console.

I'd like to ask how to solve this issue.

### Steps to Reproduce

![error](https://github.com/vercel/community/assets/118713840/07a8306d-a235-493b-8b83-44da06cd13d9)

**Comment by smaeda-ks** [comment]
Hi, could you confirm you disabled the domain lock?
https://support.google.com/domains/answer/3251178?hl=en&sjid=5727768856947167147-AP

Also, 
> If you just purchased or transferred a domain, you must wait for 60 days before you can transfer your domain to another registrar or user. This waiting period is needed from the [Internet Corporation for Assigned Names and Numbers](https://support.google.com/domains/answer/4544245) ([ICANN](https://www.icann.org/)).

**Reply by jessi0419kim** [reply]
Yes. I have unlocked the domain lock.

<img width="684" alt="error" src="https://github.com/vercel/community/assets/118713840/72b9b3e6-83d5-48bc-8b7f-856f3f031b81">


In addition, this domain was purchased about 1 years ago. So, I don't think 60 days limitation have somethig to do with the error.

**Reply by smaeda-ks** [reply]
Alright, could you contact Google Domains and see if they can regenerate a new auth code? @jessi0419kim

**Reply by jessi0419kim** [reply]
@smaeda-ks  Thank you for the suggestion. I'll try. But, Whenever I lock the domain and unlock again, It generate the new auth code automatically. So, I'm not sure if the error triggered by auth code.

Anyway, I'll ask to Google. 
Thanks.
